resources/api/kio-api.yaml

```diff
@@ -580,12 +605,6 @@ definitions:
         minimum: 1 # Tier-1
         maximum: 4 # not relevant
         example: 2
-      required_approvers:
-        type: integer
-        minimum: 1
-        description: DEPRECATED! WILL BE REMOVED IN NEXT RELASE! Minimum number of approvers needed for a version. Used by fullstop.
-        example: 2
-        default: 2
```

Comment:
This imposes the low risk that we break some consumers not conforming to the latest API doc. I'm wondering whether this is an unnecessary risk we should not take now. We have a high number of other deprecated features that are not removed yet and we are not prepared to support this deprecation now.